.github/workflows/windows.yml

```diff
@@ -79,7 +79,6 @@ jobs:
           path: _build/install.log
 
       - name: check
-        continue-on-error: true
         run: |
           cd _build/tests
           make check TESTSUITEFLAGS="--jobs=$(($(nproc)+1))" || (echo "not all tests passed")
```

Comment:
at least if `make test` fails, then _build/tests/cobol85/summary.log should be uploaded, too; in general (_maybe_ not for macos if we can't fix the testsuite otherwise) `make install` should be done only after the tests are run

Reply:
Rechecked, that's still identical and I guess that's to upload the testsuite.
@lefessan I suggest to check https://stackoverflow.com/questions/58858429 for options to make the upload different, for example using `if: always()` there, while still letting the test part fail (something similar likely occurs to the other workflows)